torch/csrc/jit/interpreter.cpp

```diff
@@ -659,17 +661,23 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     size_t last = instructions.size();
 
     while (pc < last) {
-        // std::cout << "executing " << pc << ": ";
-        // function->dumpInstruction(std::cout, pc);
-        // std::cout << "\n";
+        if(Code::debug)
+        {
```

Comment:
Can you fix the formatting to match the surround style?
```
if (Code::debug) {
```

Reply:
Ok I will do it in a few days when I have some time.